bundles/org.openhab.binding.livisismarthome/src/main/java/org/openhab/binding/livisismarthome/internal/manager/FullDeviceManager.java

```diff
@@ -0,0 +1,224 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.livisismarthome.internal.manager;
+
+import static org.openhab.binding.livisismarthome.internal.LivisiBindingConstants.BATTERY_POWERED_DEVICES;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.livisismarthome.internal.client.LivisiClient;
+import org.openhab.binding.livisismarthome.internal.client.api.entity.capability.CapabilityDTO;
+import org.openhab.binding.livisismarthome.internal.client.api.entity.capability.CapabilityStateDTO;
+import org.openhab.binding.livisismarthome.internal.client.api.entity.device.DeviceDTO;
+import org.openhab.binding.livisismarthome.internal.client.api.entity.device.DeviceStateDTO;
+import org.openhab.binding.livisismarthome.internal.client.api.entity.link.LinkDTO;
+import org.openhab.binding.livisismarthome.internal.client.api.entity.location.LocationDTO;
+import org.openhab.binding.livisismarthome.internal.client.api.entity.message.MessageDTO;
+import org.openhab.binding.livisismarthome.internal.client.exception.ApiException;
+import org.openhab.binding.livisismarthome.internal.client.exception.AuthenticationException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author Sven Strohschein - Initial contribution (but only created by refactoring the LivisiClient class)
```

Comment:
I think this will trigger a notice in the SAT report because expected message for first author is "Initial contribution".

Reply:
solved